full/atm_land_ice_flux_exchange.F90

```diff
@@ -72,7 +72,10 @@ module atm_land_ice_flux_exchange_mod
 !! FMS
 use FMS
 use FMSconstants, only: rdgas, rvgas, cp_air, stefan, WTMAIR, HLV, HLF, Radius, &
-                        PI, CP_OCEAN, WTMCO2, WTMC, EPSLN, GRAV
+                        PI, CP_OCEAN, WTMCO2, WTMC, EPSLN, GRAV, WTMH2O
+
+!gex
```

Comment:
The comment should be removed or expanded.  It doesn't provide any useful information.